CMSgov/ato-blueprint
# Feature/new projects page

Story: https://jiraent.cms.gov/browse/ISPGBSS-842

### Changes
- added a new projects page to handle projects created with the new project form
- added tests for the projects page

### Testing
- visit `/packages` to confirm it shows the number of projects created via the NEW project form (`/packages/create`) by your logged in user
- confirm the page looks the same as the previous /projects page (except for the percentage bar on the project cards)
- verify pagination works as expected
- verify total number of projects displays the correct number in the subtitle text
- run the automated tests by running `docker exec -it govready-q-dev bash` and then running the tests. (new tests can be run via `./manage.py test project.tests.test_views.ProjectListViewTest`)

_Note: we are holding on the percent bar until we can do the work necessary for the controls model (new, existing, or whatever that is)_
